.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E203, E266, E501, W503, W601
+ignore = E203, E266, E501, W503, W601, C901
```

Comment:
C901 is about too complex method, will check how to simplify the dict to yaml conf translator

Reply:
Fixed